astropy/visualization/wcsaxes/coordinate_helpers.py

```diff
@@ -207,6 +207,16 @@ def set_coord_type(self, coord_type, coord_wrap=None):
 
         self.coord_type = coord_type
 
+        if coord_wrap is not None:
+            if not isinstance(coord_wrap, u.Quantity):
+                # import pdb; pdb.set_trace()
+                warnings.warn(
+                    "Passing 'coord_wrap' as a number is deprecated. Use a Quantity with units convertible to angular degrees instead.",
+                    AstropyDeprecationWarning,
+                )
```

Comment:
Should convert the number to Quantity after warning, no?

Reply:
No, it stays as a bare number if passed as a bare number.

Reply:
As per https://github.com/astropy/astropy/pull/14050#discussion_r1043903817, I've now updated this to convert the number to a Quantity after warning.